### PR TITLE
Docker Quick Start

### PR DESCRIPTION
**Pull request by rtlechow**
* Make `docker-compose up -d` work without modifications, for an easy
  development environment.
* Setup faster with a slimmer php:alpine base image, and pull app image
  from Docker Hub.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,23 +58,22 @@ bin/cake server
 
 Note that this should never be used for a production site!
 
-### Experimental Docker Alternative
+### Docker Quick Start (Experimental!)
 
-The included Dockerfile and docker-compose.yaml files are new and will hopefully make it quicker and easier to rapidly test out the application on your own hardware.  **The Docker implementation should be considered experimental until further testing can be completed.**  If you have any issues or suggestions for simplifying deployment with Docker, or encounter any issues with deploying Zuluru under Docker, please create an issue here or at the https://github.com/pl77/Zuluru3 docker fork so it can be investigated.  
+Run `docker-compose pull && docker-compose up -d` and open
+http://localhost/installer/install. The database credentials are `host: db`,
+`username: zuluru`, `password: userpassword`. This provides a quick and easy
```
Comment (rtlechow) [inline README.md]: Can the database configuration be automated or at least pre-populated to eliminate this step?

File: README.md [hunk 2]
```diff
@@ -57,23 +58,22 @@ bin/cake server
 
 Note that this should never be used for a production site!
 
-### Experimental Docker Alternative
+### Docker Quick Start (Experimental!)
```
Comment (rtlechow) [inline README.md]: Should this be moved under `Development`? It's awkward because the "Advanced" subheading suggests it's _not_ for development.

**Comment by rtlechow** [comment]
cc: @pl77 LMK what you think, especially moving around the "advanced" bits.

**Comment by Zuluru** [comment]
Holding off merging this until any other comments come in.

**Comment by pl77** [comment]
Sorry for the delayed response, but I love the work you've done, @rtlechow .  It's an excellent separation between a simple dockerfile for taking a peak at the program and a more advanced production-level implementation with proper HTTPS.  I vote merge!

File: README.md (hunk 1 above)
Comment (pl77) [reply README.md]: I stumbled over this as well.  I'm not overly familiar with php so I wasn't going to attempt to replicate those steps in bash.

File: README.md (hunk 2 above)
Comment (pl77) [reply README.md]: I would agree.  It's a little clunky in the middle there and the readme would probably flow best if we stuck the entire docker chunk near the bottom by itself.

**Pull request merged**